android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/BdsService.java

```diff
@@ -100,17 +100,21 @@ private RequestResponse handleException(HttpURLConnection urlConnection,
   }
```

Comment:
we should be careful about     byte[] input = body.getBytes(StandardCharsets.UTF_8);, I think it will crash in api<19